lib/matplotlib/backends/backend_agg.py

```diff
@@ -460,9 +460,14 @@ def draw(self):
         # acquire a lock on the shared font cache
         RendererAgg.lock.acquire()
 
+        toolbar = self.toolbar
         try:
+            if toolbar:
+                toolbar.set_cursor(cursors.WAIT)
```

Comment:
Yes, It applies only in the sense that the code here has already been slated for removal.  Only a partial answer to your question.